### DISCUSSION TITLE
Add an organization to a private repository

### DISCUSSION THREAD

**Discussion by aaly-dp** [community/community#23526] [New to GitHub]
<p>Hello,<br>
We created an Github orgaization in our company. We now want to configure for the following Use Case:<br>
External Repository can add our organization as a collaborator, so that every user in our organization can be part of the external repository.</p>
<p>How can I do that?</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="aaly-dp" data-post="1" data-topic="247596">
<div class="title">
<div class="quote-controls"></div>
 aaly-dp:</div>
<blockquote>
<p>External Repository</p>
</blockquote>
</aside>
<p>who owns that external repo ? is it related to your Org ? then you can request a transfer to the Org, if you are on the same purpose,</p>
<p><a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository">https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository</a></p>
<p>or maybe you can fork the repo to that Org, so you will have a copy and thereby easily shared to the members</p>

**Comment by aaly-dp** [comment]
<p>Hallo,<br>
The repository belongs to a 3rd-party and we want to collaborate with them.</p>

**Comment by jfullstackdev** [comment]
<p>but it’s private ? meaning, at least one member knows the exact repo ? it’s best you contact them of what will be the agreement,</p>
<p>what I’m seeing, since it’s private, is to transfer it to the Org</p>

**Comment by aaly-dp** [comment]
<p>OK let me rephrase. Our Use Case is, that organisations with an existing repository are in need of development from our organisations in specific ares. So we want to be able to add our organisations as a contributor in an external repository. I tried this with a private repository of my own to add the organisation as a contributor, but sadly I couldn’t select/find the organisation.</p>
<p>Thanks for all your answers until now and best regards</p>

**Comment by jfullstackdev** [comment, marked as answer]
<aside class="quote no-group" data-username="aaly-dp" data-post="5" data-topic="247596">
<div class="title">
<div class="quote-controls"></div>
 aaly-dp:</div>
<blockquote>
<p>So we want to be able to add our organisations as a contributor in an external repository.</p>
</blockquote>
</aside>
<p>as we mentioned above, if it’s a public repo, simply fork the external repo to have the Org’s copy</p>
<p>but this one, if this is private, then I think, they need to manually add you one by one,</p>
<p>so, this might be a feature request, but there is no assurance, try requesting it at the Product Team</p>
<p><a href="https://github.com/github/feedback">https://github.com/github/feedback</a></p>
<p>I’m not very sure if there is a workaround in the paid subscriptions, so yes, you may try requesting this one for GitHub Free,</p>
<p>even in the past, we could not add unlimited collaborators in a private repo but now we already have it</p>
